### PR TITLE
Convert remaining apps from build() -> generate()

### PR DESCRIPTION
**Pull request by steven-johnson**

### CODE REVIEW

File: apps/linear_algebra/src/blas_l1_generators.cpp [hunk 1]
```diff
@@ -64,25 +67,22 @@ class AXPYGenerator :
...
+        result_(tail) = calc(tail);
 
         if (vectorize_) {
             Var ii("ii");
-            result.update().vectorize(vecs, vec_size);
+            result_.update().vectorize(vecs, vec_size);
         }
 
-        result.bound(i, 0, x_.width());
-        result.output_buffer().dim(0).set_bounds(0, x_.width());
```
Comment (abadams) [inline apps/linear_algebra/src/blas_l1_generators.cpp]: Very happy to see all this output_buffer() nonsense go away

File: apps/linear_algebra/src/blas_l2_generators.cpp [hunk 2]
```diff
@@ -155,21 +157,18 @@ class GEMVGenerator :
             A_.dim(0).set_min(0).dim(1).set_min(0);
             x_.dim(0).set_bounds(0, A_.height());
             y_.dim(0).set_bounds(0, A_.width());
-            result.output_buffer().dim(0).set_bounds(0, A_.width());
         }
 
-        Func output("output");
-        output(i) = result(i);
+        // TODO: is this necessary? Doesn't result already have the necessary specializations?
```
Comment (abadams) [inline apps/linear_algebra/src/blas_l2_generators.cpp]: Looks like it's a workaround from: 

https://github.com/halide/Halide/commit/cf999bf71939261bdcbb92d87fc4d07db5770732

We probably have the tools to deal with this properly now. I'd add a TODO: delete this pointless memcpy

**Comment by abadams** [comment]
lgtm

**Comment by steven-johnson** [comment]
Windows failures are unrelated.

**Pull request merged**